src/registry.jl

```diff
@@ -16,6 +16,13 @@ end
 
 add_format(format"RData", detect_rdata, [".rda", ".RData", ".rdata"], [:RData, LOAD])
 
+add_format(format"CSV", (), [".csv"], [:CSVFiles])
```

Comment:
See my other comment, essentially the CSVFiles package exists because there are multiple potential target formats for a CSV file and there are multiple packages that do CSV reading.